File: backend/src/main/java/com/simats/neoomfs/dto/request/DentalExaminationRequest.java
package com.simats.neoomfs.dto.request;

import lombok.Data;

/**
 * Inbound DTO for dental examination (Step 6 of the wizard).
 */
@Data
public class DentalExaminationRequest {
    private String asaClass;
    private String pellGregoryClass;
    private String winterClassification;
    private String upperThirdMolar;
    private Integer difficultyScore;
    private Integer mouthOpeningMm;
    private String oralHygieneStatus;
    private String periodontalStatus;
    private boolean activeInfection;
    private boolean swelling;
    private boolean trismus;
    private String toothNumber;
    private String clinicalExaminationNotes;
}

File: frontend/app/src/main/java/com/simats/neoomfs/models/DentalExaminationRequest.kt
package com.simats.neoomfs.models

data class DentalExaminationRequest(
    val asaClass: String?,
    val pellGregoryClass: String?,
    val winterClassification: String?,
    val upperThirdMolar: String?,
    val difficultyScore: Int?,
    val mouthOpeningMm: Int?,
    val oralHygieneStatus: String? = null,
    val periodontalStatus: String? = null,
    val activeInfection: Boolean,
    val swelling: Boolean,
    val trismus: Boolean,
    val toothNumber: String?,
    val clinicalExaminationNotes: String? = null
)
